Sources/FoundationInternationalization/Locale/Locale_ICU.swift

```diff
@@ -62,47 +62,62 @@ internal final class _LocaleICU: _LocaleProtocol, Sendable {
         var collationIdentifierDisplayNames: [String : String?] = [:]
         var currencySymbolDisplayNames: [String : String?] = [:]
         var currencyCodeDisplayNames: [String : String?] = [:]
+        var numberFormatters = NumberFormattersBox()
```

Comment:
This resolves a concurrency warning by putting the unsafe mutable pointers into an `@unchecked Sendable` box with some explanation on why this is safe. Using the `@unchecked Sendable` box instead of marking this whole state as `@unchecked Sendable` ensures that any new state added is safely `Sendable`